ElectronClient/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -54,28 +93,69 @@ export default function NoteContentPropertiesDialog(props:NoteContentPropertiesD
 		props.onClose();
 	};
 
-	const createItemField = (key: string, value: number) => {
-		const labelComp = <label style={Object.assign({}, theme.textStyle, theme.controlBoxLabel)}>{keyToLabel[key]}</label>;
-		const controlComp = <div style={Object.assign({}, theme.textStyle, theme.controlBoxValue)}>{value}</div>;
+	const labelCompStyle = {
+		...theme.textStyle,
+		fontWeight: 'bold',
+		width: '10em',
+	};
+
+	const controlCompColumnWidth = '7em';
+
+	const controlCompStyle = {
+		...theme.textStyle,
+		width: controlCompColumnWidth,
+		textAlign: 'center',
+	};
+
+	const createTableBodyRow = (key: string, value: number, strippedValue: number) => {
+		const labelComp = <td style={labelCompStyle}>{keyToLabel[key]}</td>;
+		const controlComp = <td style={controlCompStyle}>{value}</td>;
+		const strippedControlComp = <td style={controlCompStyle}>{strippedValue}</td>;
 
 		return (
-			<div key={key} style={theme.controlBox} className="note-text-property-box">{labelComp}{controlComp}</div>
+			<tr key={key}>{labelComp}{controlComp}{strippedControlComp}</tr>
 		);
 	};
 
+	const tableHeaderStyle = {
+		...theme.textStyle,
+		width: controlCompColumnWidth,
+		textAlign: 'center',
+	};
+
+	const tableHeader = (
+		<tr>
+			<th style={tableHeaderStyle}></th>
+			<th style={tableHeaderStyle}>{_('Raw Source')}</th>
+			<th style={tableHeaderStyle}>{_('Markdown-stripped source')}</th>
```

Comment:
I would use `Editor` and `Viewer` here instead. (The same terms used for the layout.)

Also, what happens, if you click `Content properties` when you are in

- editor only mode
- preview only mode
- WYSIWYG editor

Reply:
Changed the column headers as you instructed.
The behaviour doesn't change depending on the layout. The idea of the PR is to provide the user a better idea as to how many lines, characters he/she has typed and what actually will be shown on screen.

- Both editor and viewer:
![Screenshot from 2020-04-14 18-43-06](https://user-images.githubusercontent.com/54888685/79228930-db18bf80-7e7f-11ea-9656-5a332a3277c5.png)

- Only editor:
![Screenshot from 2020-04-14 18-46-25](https://user-images.githubusercontent.com/54888685/79229226-495d8200-7e80-11ea-9424-f5132efa81a5.png)


- Only preview:
![Screenshot from 2020-04-14 18-49-57](https://user-images.githubusercontent.com/54888685/79229518-c7ba2400-7e80-11ea-8ad6-59b43f6db4a2.png)

As for the WYSIWYG editor, there is apparently no content properties button. So this feature as of now does not apply to the WYSIWYG editor.
![Screenshot from 2020-04-14 18-53-00](https://user-images.githubusercontent.com/54888685/79229791-38614080-7e81-11ea-9006-8a069dfc494b.png)

Reply:
Thanks for the info.

I just wanted to make sure that it still worked in all layout modes.